data/campaigns/Eastern_Invasion/ART_MUSIC_LICENSE.txt

```diff
@@ -0,0 +1,307 @@
+
+AUDIO
+===============
+
+MUSIC
+---------------
+
+cry from elensefar (pluft) - CC BY-SA 4.0
+    https://forums.wesnoth.org/viewtopic.php?t=48565
+    https://soundcloud.com/pluft/cry-from-elensefar
+
+heavens (gabriel silver), remixed by dalas - CC BY-SA 4.0 (both)
+    https://forums.wesnoth.org/viewtopic.php?t=51483
+    https://www.youtube.com/watch?v=AOvwWOGH7bQ&list=PLk-SdLbgd5ol62NX0q7ys5ZbmJFVAMzCF&index=2
+
+
+SOUNDS
+---------------
+
+wind (hackerb9) - CC 0
+    edited by dalas; I release the edited sound under CC BY-SA 4.0
+    https://freesound.org/people/hackerb9/sounds/555246/
+
+gate-fall
+lich_die_wail
+magic-twilight
+magic-twilight-miss
+    made by dalas by modifying pre-existing wesnoth sounds (GPL v2.0 or later)
+
+warhammer (Iwiploppenisse), remixed by dalas (with pre-existing wesnoth sounds) - CC BY-SA 4.0
+    https://freesound.org/people/Iwiploppenisse/sounds/156031/
```

Comment:
We can't use this, it's CC BY-NC 4.0 (probably, maybe not even that).

The credits on the page that this links to say it's made from the three other sounds, which are under these licenses:
* CC Sampling+ 1.0 https://freesound.org/people/ljudman/sounds/33245/
* CC BY-NC 4.0 https://freesound.org/people/HerbertBoland/sounds/33637/
* The third sound that it's made from, https://freesound.org/people/themfish/sounds/34202/ claims to be CC BY 4.0, but itself also uses the CC BY-NC 4.0 sound from HerbertBoland.

A general note about this file: for the CC BY and CC BY-SA licenses, I think licensees are meant to copy the source's attribution notice, so I'd expect this text from Iwiploppenisse's page to be copied into this file:

> Made by mixing together: CinematicBoomNorm, glashouse and grenade.
>
> This sound is free to use wherever you'd like. But if you still want to support me, buy me a coffee! https://www.buymeacoffee.com/I2XxQGp
>
> GRENADE: "Made" By Ljudmann: http://freesound.org/people/ljudman/sounds/33245/
>
> GLASSHOUSE: "Made" By Themfish: http://freesound.org/people/themfish/sounds/34202/
>
> CINEMATICBOOMNORM: Made by HerbertBoland: http://freesound.org/people/HerbertBoland/sounds/33637/

Please say when the licensing for the rest of the material has been audited, but having spotted this I'm stopping looking until then.